src/views/Samples/Page.tsx

```diff
@@ -0,0 +1,73 @@
+import topArrow from "images/arrow-down.svg";
+import bottomArrow from "images/arrow-up.svg";
+import React from "react";
+import { Link } from "react-router-dom";
+import PageHeader from "../../components/Header/PageHeader";
+import { SubHeader } from "../../components/Collection/SubHeader";
+import "./scss/index.scss";
+
+const Page = props => {
+  const { data, history } = props;
+  const handleBack = () => {
+    history.push("/");
+  };
+  const regex = /\s|_|(?=[A-Z])/;
+
+  return (
+    <div className="divImg sample-page">
+      <PageHeader
+        back={true}
+        cart={true}
+        search={true}
+        handleClick={handleBack}
+      />
+
+      <div className="browse-cabinet__heading">
+        <h3>Samples</h3>
+      </div>
```

Comment:
This should be inside: `SubHeader`, and expose a prop: `title="Samples"`

It seems like it's the same UI section.

Reply:
Okay